tests/integration/cli/test_compile.py

```diff
@@ -19,8 +19,12 @@ def test_missing_extensions(ape_cli, runner, project):
     assert ".test" in result.output
     assert ".foobar" in result.output
 
+
+@skip_projects_except(["unregistered-contracts"])
+def test_no_compiler_for_extension(ape_cli, runner, project):
     result = runner.invoke(ape_cli, ["compile", "contracts/Contract.test"])
-    assert result.exit_code == 0
+    # TODO: FIX
+    # assert result.exit_code == 0
```

Comment:
Instead of doing this, I would use `@pytest.mark.xfail(reason="...")`, or just let it fail for now inside of your PR if you intend to fix it.

By doing this, we run the danger of forgetting that the test case is actually broken.